### DISCUSSION TITLE
ONLY USE DISCUSSIONS WITH LOW PRIORITY QUESTIONS/REQUESTS

### DISCUSSION THREAD

**Discussion by ClassicOldSong** [ClassicOldSong/Apollo#7] [Announcements]
Use issues if possible since discussions are not likely to notify users in time.

**Comment by Killemfirstone** [comment]
Dear ClassicOldSong,

got all runnig and thanks for your impressing work with multiple Apollo instances and Powershell.

I have only one questions, how can i run this powershell script automaticly when windows starts up?

I wont work via autostart folder and over shedule tasks.  

Already try to build a batch file, but dont get it running. Maybe you have an solution on Hand for this? It would be fine to start the Host PC via WOL and the scripts will run automaticly to present multiple Apollo instances after start. :)

**Reply by Killemfirstone** [reply]
BTW. all runs on WIN 11 Pro..

**Reply by ClassicOldSong** [reply]
No, unless you disable UAC completely, then you can put the script in the AutoStart folder.

**Reply by Killemfirstone** [reply]
Hi, 

UAC is already disabled but autostart neverless didnt work.

**Reply by ClassicOldSong** [reply]
You need to set the script to always run with Administrator privilege.

**Reply by vitorhugoaranda** [reply]
> Dear ClassicOldSong,
> 
> got all runnig and thanks for your impressing work with multiple Apollo instances and Powershell.
> 
> I have only one questions, how can i run this powershell script automaticly when windows starts up?
> 
> I wont work via autostart folder and over shedule tasks.
> 
> Already try to build a batch file, but dont get it running. Maybe you have an solution on Hand for this? It would be fine to start the Host PC via WOL and the scripts will run automaticly to present multiple Apollo instances after start. :)

I use a smart plug with alexa/google, so I send command start computer or widgets in cellphone, for me is a good solution, because works outside your LAN, from internet, only need config bios to always turn on if energy power is on.

![{87176CEF-C070-4DFE-96AF-4A24A98EDCB8}](https://github.com/user-attachments/assets/3faa5a13-913e-4348-8086-768498201b16)

**Comment by Killemfirstone** [comment]
Hi, starting the Computer works fine with wol and Alexa, since years.The Problem ist to Autorun Apollo with the multiple  instance Script. I have all done uac ist Off and Powershell ist Set to run via Admin. But it will Not Autostart after booting the machine.I dont get it work.Manually start works Like a Charme.Michael KellerCombrinkstrasse 25a59229 DolbergVon: vitorhugoaranda ***@***.***>Gesendet: Montag, März 10, 2025 2:32:12 PMAn: ClassicOldSong/Apollo ***@***.***>Cc: Killemfirstone ***@***.***>; Comment ***@***.***>Betreff: Re: [ClassicOldSong/Apollo] ONLY USE DISCUSSIONS WITH LOW PRIORITY QUESTIONS/REQUESTS (Discussion #7)

Dear ClassicOldSong,
got all runnig and thanks for your impressing work with multiple Apollo instances and Powershell.
I have only one questions, how can i run this powershell script automaticly when windows starts up?
I wont work via autostart folder and over shedule tasks.
Already try to build a batch file, but dont get it running. Maybe you have an solution on Hand for this? It would be fine to start the Host PC via WOL and the scripts will run automaticly to present multiple Apollo instances after start. :)

I use a smart plug with alexa/google, so I send command start computer ou widgets in cellphone, for me is a good solution, because works outside your LAN, from internet, only need config bios to always turn on if energy power is on.
87176CEF-C070-4DFE-96AF-4A24A98EDCB8.png (view on web)

**Reply by ClassicOldSong** [reply]
You need to run manually for multiple instances, or you'll need to disable UAC completely for it to be able to auto start.

**Comment by BobBobalong** [comment]
Hello 

Thank you for an amazing fork I have been using it up till now so stream to my s25 ultra with no issues.

I have now gone over to a windows based handheld MSI Claw 8 ai there use of touch screen control is limiting.

My question is the touchpad control for the mouse control you implement into your build is flawless is this a possibility as a standalone as many who have thease new devices rog ally xbox x and msi claw ect would benifit greatly.

Thank you

**Reply by ClassicOldSong** [reply]
Moonlight for desktop is almost a completely different app. I don't have time to make any change to it yet

**Reply by BobBobalong** [reply]
Thank you for the reply 

I completely understand as the additional work that would be involved would be a deviation from your project.

Thank you
